tests/queries/0_stateless/02875_show_functions.sh

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env bash
+# Tags: no-parallel
+# Because other tests may create UDFs
```

Comment:
Can we use clickhouse local instead and keep it parallel?

Reply:
Thanks, this will be much better!